GameServerLib/Logic/Scripting/Engine/GameScriptBuffController.cs

```diff
@@ -8,25 +8,26 @@
 
 namespace LeagueSandbox.GameServer.Logic.Scripting.CSharp
 {
-    public class BuffGameScriptController
+    public class GameScriptBuffController
     {
-        Unit _unit;
-        BuffGameScript _gameScript;
+        Unit _unit, _target;
```

Comment:
Could the `_unit` variable be renamed to something more clear? Is it the owner/creator of the buff or something else? Probably should be renamed `_owner`